tree/ntuple/v7/inc/ROOT/RNTupleReader.hxx

```diff
@@ -293,12 +293,36 @@ public:
       return GetView<T>(RetrieveFieldId(fieldName), objPtr);
    }
 
+   template <typename T>
+   ROOT::RNTupleView<T> GetView(std::string_view fieldName, std::shared_ptr<T> objPtr, const std::type_info &ti)
```

Comment:
I just noticed we don't have any documentation on the `GetView` methods that accept a pointer...maybe we should add it (even just to one of them and point the other methods' doc to it)

Reply:
See #18261 for the already existing ones, will do the same for these new additions afterwards!